test/test-pmctl-local.js

```diff
@@ -50,162 +160,139 @@ helper.manager = function manager(callback) {
   return pm;
 };
 
-helper.pmctl = {};
-
-// Wait on cmd to write specific output
-helper.pmctl.waiton = waiton;
-function waiton(cmd, output) {
-  while (true) {
-    try {
-      console.trace('WAITON %s', cmd);
-      expect(cmd, output);
-      return;
-    } catch(er) {
-      pause();
-    }
-  }
+function test(name, callback) {
+  // XXX could skip tests here, based on presence of TAP_ONLY
+  tap.test(name, function(t) {
+    console.log('TEST:', name);
+    // XXX could set t globally here, so it doesn't have to be an explicit
```

Comment:
Yes, T would always be the enclosing tap.test()'s t, so it becomes an implicit param of expect(), etc. Just shortens the tests, slightly. Probably I won't bother.